proto/proto/wallet.proto

```diff
@@ -0,0 +1,62 @@
+syntax = "proto3";
+package penumbra.wallet;
+
+import "transaction.proto";
+import "crypto.proto";
+
+service WalletProtocol {
+  // TODO: do we want to include auth/signatures in all requests here?
+  /// Get current status of chain sync
+  rpc Status() returns (StatusResult);
+  /// Create a new transaction
+  rpc CreateTransaction(CreateTransactionRequest) returns (CreateTransactionResult);
+  // TODO: not sure what listing transactions would actually look like
+  /// List pending transactions
+  rpc Transactions(TransactionsRequest) returns (stream transaction.Transaction);
+  // TODO: is this UtxoResult analogous to transaction.Output? or UnspentNote? should we use that here?
+  /// List unspent transaction outputs
+  rpc Utxo(UtxoRequest) returns (stream UtxoResult);
+  /// Display current wallet balance
+  rpc Balance(BalanceRequest) returns (stream BalanceResult);
+}
+
+// Requests the wallet server to sign and submit a transaction.
+message CreateTransactionRequest {
+    transaction.TransactionBody body = 1;
+}
+
+// Returns whether the creation of the transaction was successful or not.
+message CreateTransactionResult {
+    bool success = 1;
+}
+
+// Returns the status of the wallet server and whether it is synchronized with the chain state.
+message StatusResult {
+  /// Whether the wallet service is synchronized with the chain state.
+  bool synchronized = 1;
+}
+
+// Requests all transactions associated with the wallet.
+message TransactionsRequest {
+    uint64 id = 1;
+}
+
+// Requests all unspent transactions associated with the wallet.
+message UtxoRequest {
+    uint64 id = 1;
+}
+
+// Contains an unspent transaction.
+message UtxoResult {
+}
```

Comment:
It wasn't clear what this would actually look like or what the analog in the current pcli implementation is

Reply:
The analog would be unspent notes, which are currently visible in `pcli` only through `balance --by-note` but otherwise aren't exposed.